src/find/matchers/mod.rs

```diff
@@ -692,6 +692,20 @@ fn build_matcher_tree(
 
                 return Ok((i, top_level_matcher.build()));
             }
+            // In our implementation, including the `-exec` parameter,
+            // it is always run in a single thread.
+            // Therefore, there is no race condition for now.
```

Comment:
The "race condition" that this flag is talking about is *another* command simultaneously modifying the directory while `find` is reading it, something like

```console
tavianator@graphene $ touch {1..10000}
tavianator@graphene $ find -size 1 & rm ./*
[1] 65991
find: ‘./10000’: No such file or directory
[1]  + 65991 exit 1     find -size 1
tavianator@graphene $ touch {1..10000}
tavianator@graphene $ find -ignore_readdir_race -size 1 & rm ./*
[1] 66034
[1]  + 66034 done       find -ignore_readdir_race -size 1
```

Reply:
Sorry, I forgot about this. The race condition does happen when deleting a large number of files, and I'm rewriting a piece of code that implements this feature.

Reply:
Terribly sorry, I did misunderstand the meaning of `readdir_race`. 

I rechecked the code and read the GNU documentation. The `-ignore_readdir_race` parameter requires a way to suppress file not found errors globally in the software, so I need to complete https://github.com/uutils/findutils/issues/15 to centralize the error handling logic before I can return here. 

I will finish them as soon as possible. :)